spec/core/agent_plugin_manager_spec.rb

```diff
@@ -0,0 +1,36 @@
+# encoding: utf-8
+require "spec_helper"
+require "logstash/runner"
+require "stud/task"
+
+describe LogStash::AgentPluginManager do
+  class TestAgent < LogStash::Agent; end
+  class TestAgent2 < LogStash::Agent; end
+
+  subject { described_class }
```

Comment:
for the way you use subject, I would recommend actually using the sintax ``` subject(:foo) { Klass.new } ``` like this you don't need to use subject and can use :foo as a more contextualized var name.